webpack.dev.babel.js

```diff
@@ -9,7 +9,8 @@ import webpack from 'webpack';
 export default {
   entry: {
     app: [
-      'webpack/hot/only-dev-server',
+      'webpack-dev-server/client?http://localhost:8080',
```

Comment:
Want to get this PR merged as it is blocking @davezuko.  Will fix errors in this PR and push improvements to another PR.